hapi-fees/src/main/java/com/hedera/node/app/hapi/fees/usage/crypto/CryptoApproveAllowanceMeta.java

```diff
@@ -52,16 +48,20 @@ public CryptoApproveAllowanceMeta(
             long transactionValidStartSecs) {
         effectiveNow = transactionValidStartSecs;
         msgBytesUsed = bytesUsedInTxn(cryptoApproveTxnBody);
-        cryptoAllowances = convertToCryptoMap(cryptoApproveTxnBody.getCryptoAllowancesList());
-        tokenAllowances = convertToTokenMap(cryptoApproveTxnBody.getTokenAllowancesList());
-        nftAllowances = convertToNftMap(cryptoApproveTxnBody.getNftAllowancesList());
+        cryptoAllowances =
+                CryptoContextUtils.convertToCryptoMap(
+                        cryptoApproveTxnBody.getCryptoAllowancesList());
+        tokenAllowances =
+                CryptoContextUtils.convertToTokenMap(cryptoApproveTxnBody.getTokenAllowancesList());
+        nftAllowances =
+                CryptoContextUtils.convertToNftMap(cryptoApproveTxnBody.getNftAllowancesList());
     }
 
     private int bytesUsedInTxn(CryptoApproveAllowanceTransactionBody op) {
         return op.getCryptoAllowancesCount() * CRYPTO_ALLOWANCE_SIZE
                 + op.getTokenAllowancesCount() * TOKEN_ALLOWANCE_SIZE
                 + op.getNftAllowancesCount() * NFT_ALLOWANCE_SIZE
-                + countSerials(op.getNftAllowancesList()) * LONG_SIZE;
+                + CryptoContextUtils.countSerials(op.getNftAllowancesList()) * LONG_SIZE;
```

Comment:
Can you get the imports back, so that each constant need not have `CryptoContextUtils.`